### ISSUE TITLE
Picking support

### ISSUE THREAD

**Issue by lilleyse** [CesiumGS/cesium-omniverse#154]
Implement raycast picking similar to Cesium for Unreal / Unity so that it's possible to do things like click an OSM building and view its metadata.

Requires physics support first: https://github.com/CesiumGS/cesium-omniverse/issues/153.

**Comment by timoore** [comment]
Taking a look at this and noting that it depends on #153 . Do you see using https://github.com/CesiumGS/cesium-native/pull/777 as an alternative?

**Comment by lilleyse** [comment]
> Do you see using https://github.com/CesiumGS/cesium-native/pull/777 as an alternative?

Potentially. We should check out `omni.kit.raycast.query` first which doesn't depend on physics. Not sure if it works with Fabric yet.

**Comment by lilleyse** [comment]
Raycasting request from the forum: https://community.cesium.com/t/ground-detection-in-cesium-for-omniverse/29819/2

**Comment by lilleyse** [comment]
@timoore do you know if the ray cast query returns the path of the Fabric prim? If that's not possible we'll likely need to postpone this feature until that capability is supported.

**Comment by timoore** [comment]
> @timoore do you know if the ray cast query returns the path of the Fabric prim? If it doesn't we'll likely need to postpone this feature until that capability is supported.

The raycast query result has methods for get_target_usd_path(), instance_id(), and primitive_id(). I don't know if that is all completely specific to USD or would return sensible results for a Fabric prim.
